salt/modules/status.py

```diff
@@ -1054,8 +1054,12 @@ def master(master=None, connected=True):
     if not master_ips:
         return
 
+    pub_port = None
```

Comment:
It's unclear why we're changing this logic up here instead of just down in the utils function. It seems to me it would make more sense to change it there and then we can avoid changing the arg list for `connected_masters` as well. Thoughts?